pkg/koordlet/metricsadvisor/collector_gpu.go

```diff
@@ -197,17 +205,29 @@ func (g *gpuDeviceManager) getTotalGPUUsageOfPIDs(pids []uint64) []metriccache.G
 	for i := 0; i < g.deviceCount; i++ {
 		if value, ok := tmp[i]; ok {
 			rtn = append(rtn, metriccache.GPUMetric{
-				DeviceUUID: g.devices[i].DeviceUUID,
-				Minor:      g.devices[i].Minor,
-				SMUtil:     value.SMUtil,
-				MemoryUsed: *resource.NewQuantity(int64(value.MemoryUsed), resource.BinarySI),
+				DeviceUUID:  g.devices[i].DeviceUUID,
+				Minor:       g.devices[i].Minor,
+				SMUtil:      value.SMUtil,
+				MemoryUsed:  *resource.NewQuantity(int64(value.MemoryUsed), resource.BinarySI),
+				TotalMemory: *resource.NewQuantity(int64(g.devices[i].TotalMemory), resource.BinarySI),
 			})
 		}
 	}
 	return rtn
 }
 
 func (g *gpuDeviceManager) getPodGPUUsage(podParentDir string, cs []corev1.ContainerStatus) ([]metriccache.GPUMetric, error) {
+	runingContainer := make([]corev1.ContainerStatus, 0)
+	for i := range cs {
+		if cs[i].State.Running == nil {
+			klog.V(6).Infof("non-running container %s", cs[i].ContainerID)
+			continue
```

Comment:
according to this [convention](https://github.com/koordinator-sh/koordinator/blob/main/docs/contributors/logging.md), I think it should be V(5). WDYT?